h1st/model/model.py

```diff
@@ -95,3 +109,49 @@ def process(self, input_data: Dict) -> Dict:
         """
         # not raise NotImplementedError so the initial model created by integrator will just work
         return input_data
+
+    def train(self, data: Dict[str, Any] = None) -> None:
+        """
+        Implement logic to create the corresponding MLModel, including both training and evaluation.
```

Comment:
Explain what we are doing here, which is to call the appropriate `train` or `fit` on the `base_model`, using the prepared training data.

Reply:
Moving fast.